clippy_lints/src/utils/paths.rs

```diff
@@ -90,6 +90,8 @@ pub const RANGE_TO_INCLUSIVE: [&str; 3] = ["core", "ops", "RangeToInclusive"];
 pub const RANGE_TO_INCLUSIVE_STD: [&str; 3] = ["std", "ops", "RangeToInclusive"];
 pub const RANGE_TO_STD: [&str; 3] = ["std", "ops", "RangeTo"];
 pub const RC: [&str; 3] = ["alloc", "rc", "Rc"];
+pub const REFCELL_BORROW: [&str; 4] = ["core", "cell", "RefCell", "borrow"];
+pub const REFCELL_BORROWMUT: [&str; 4] = ["core", "cell", "RefCell", "borrow_mut"];
```

Comment:
```suggestion
pub const REFCELL_BORROW_MUT: [&str; 4] = ["core", "cell", "RefCell", "borrow_mut"];
```

Reply:
Shouldn't this use diagnostics items?

Reply:
I'm not sure, how far along diagnostic items are and if these methods are already diagnostic items. I think the initial though of diag items was, to get rid of the `utils/paths.rs` module, but this wasn't followed up from the Clippy side, since adding diag items to the compiler.